src/App.js

```diff
@@ -1,19 +1,31 @@
 
+import { Route, Routes } from 'react-router-dom';
 import './App.css';
-import About from './about';
+// import About from './about';
 import Contact from './contact';
-import Faq from './faq';
+// import Faq from './faq';
 import Home from './home';
+import Navbar from './navbar';
+import Create from './create';
+import NotFound from './not-found';
+
 
 
 function App() {
   return (
 <div>
-  <p>this is the div yesterday</p>
-
-  <Home/>
-  <About/>
-  <Contact/>
+  {/* <p>this is the div yesterday</p> */}
+<Navbar/>
+<Routes>
+  <Route path="/" element= {<Home/>}/>
+  <Route path="/create" element= {<Create/>}/>
+  <Route path="/contact" element= {<Contact/>}/>
+  <Route path="*" element= {<NotFound/>}/>
+</Routes>
+  {/* <Home/> */}
+  {/* <About/> */}
+  {/* <Contact/> */}
+  {/* <Faq/> */}
```

Comment:
The current pull request will be approved but kindly ensure to remove unneeded lines before submitting the next pull request. Lines commented out may cause issues in the future. So it is ideal to remove them.